include/swift/AST/Decl.h

```diff
@@ -770,6 +774,15 @@ class alignas(1 << DeclAlignInBits) Decl {
   /// \brief Mark this declaration as implicit.
   void setImplicit(bool implicit = true) { DeclBits.Implicit = implicit; }
 
+  /// \brief Determine whether this declaration was an element in a capture
+  //  list.
+  bool isCaptureList() const { return DeclBits.CaptureList; }
+
+  /// \brief Mark this declaration as an element in a capture list.
+  void setCaptureList(bool captureList = true) {
```

Comment:
Since this only seems to be effectively read only, can this setter be removed and the bit set in the constructor?